lightning/src/util/poly1305.rs

```diff
@@ -12,25 +12,33 @@ use core::convert::TryInto;
 
 #[derive(Clone, Copy)]
 pub struct Poly1305 {
-	r         : [u32; 5],
-	h         : [u32; 5],
-	pad       : [u32; 4],
-	leftover  : usize,
-	buffer    : [u8; 16],
-	finalized : bool,
+	r: [u32; 5],
+	h: [u32; 5],
+	pad: [u32; 4],
+	leftover: usize,
+	buffer: [u8; 16],
+	finalized: bool,
```

Comment:
Candidate for a skip, and several other custom whitespaces used for alignment in this file.